cmd/cloud/workspace.go

```diff
@@ -71,6 +147,47 @@ func workspaceSwitch(cmd *cobra.Command, out io.Writer, args []string) error {
 	return workspace.Switch(id, astroClient, out)
 }
 
+func addWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
```

Comment:
should this be lower cased and validated as an email before using it?

Reply:
It seems that emails are always stored as lower case. Not sure if we need to validate as an email. if someone gives a non email the API will give an error